docs/_includes/tests/dialog-search.html

```diff
@@ -1,20 +1,10 @@
----
-ds: 4
-layout: test
-title: Search
----
-<svg hidden>
-    <symbol id="svg-icon-search" viewBox="4 0 32 32">
-        <path d="M33.327 30.205l-6.79-6.79c1.561-1.873 2.498-4.293 2.498-6.946-0.078-6.010-4.995-11.005-11.083-11.005s-11.005 4.995-11.005 11.005 4.917 11.083 11.005 11.083c2.732 0 5.151-1.015 7.102-2.576l6.79 6.79c0.234 0.234 0.468 0.312 0.702 0.312s0.546-0.078 0.702-0.312c0.468-0.468 0.468-1.171 0.078-1.561zM9.054 16.468c0-4.917 3.98-8.898 8.898-8.898s8.898 3.98 8.898 8.898-3.98 8.976-8.898 8.976c-4.917 0-8.898-3.98-8.898-8.976z"></path>
-    </symbol>
-</svg>
 <div class="test">
     <div role="dialog" class="dialog" id="dialog" aria-label="Search">
         <div class="dialog__window dialog__window--full" role="document">
             <div class="dialog__search">
                 <form>
                     <button type="submit">
-                        <svg aria-label="Search">
+                        <svg class="icon icon--search" aria-label="Search">
```

Comment:
Why aren't we moving `aria-label="Search"` onto `button` like we are in other tests?

Reply:
This page is sort of unmaintained. It's an example I ported over from another place.

@ianmcburnie Should I just remove this one?

Reply:
Yes, aria-label should go on the button. In general, some of the ARIA on the test pages is a bit inconsistent (mainly due to all the copy & paste that has occurred over the years). It's good to fix up where we see it though.

Reply:
@ianmcburnie I mean, should I remove this `dialog-search.html` test page?

Reply:
Yes, if it's not part of DS6